sessionctx/variable/tidb_vars.go

```diff
@@ -989,6 +989,12 @@ const (
 	// TiDBTTLRunningTasks limits the count of running ttl tasks. Default to 0, means 3 times the count of TiKV (or no
 	// limitation, if the storage is not TiKV).
 	TiDBTTLRunningTasks = "tidb_ttl_running_tasks"
+	// EnableRuntimeFilter control runtime filter, true: runtime filter open.
+	EnableRuntimeFilter = "enable_runtime_filter"
+	// RuntimeFilterTypeName the value of is string, a runtime filter type list split by ",", such as: "IN,MIN_MAX"
+	RuntimeFilterTypeName = "runtime_filter_type"
+	// RuntimeFilterModeName the mode of runtime filter, such as "LOCAL"
+	RuntimeFilterModeName = "runtime_filter_mode"
```

Comment:
TiDB-specific variables almost always need a "tidb_" prefix.
And I'm not sure but maybe we need to ask PM whether we need a "tidb_opt_" prefix here.

Reply:
RuntimeFilter is not only a tidb function but also a tiflash function. So should we add this prefix ?

Reply:
I thought we should, because we have `tidb_enable_mpp`, `tidb_mpp_store_fail_ttl`, and `tidb_max_tiflash_threads`, but then I found now we also have `mpp_exchange_compression_mode` and `mpp_version`.
So now I'm also not 100% sure 😂

Reply:
Yes. So what exactly are our rules for adding prefixes? 😂